Xamarin.Forms.Platform.Android/Renderers/TimePickerRenderer.cs

```diff
@@ -92,7 +93,8 @@ void OnClick()
 			TimePicker view = Element;
 			ElementController.SetValueFromRenderer(VisualElement.IsFocusedPropertyKey, true);
 
-			_dialog = new TimePickerDialog(Context, this, view.Time.Hours, view.Time.Minutes, false);
+			bool is24HourFormat = DateFormat.Is24HourFormat(Context);
```

Comment:
I suppose it's a matter of preference. You're calling a method named `Is24HourFormat` and declaring a variable with the same name. From the method name , we could infer the same thing.